var/spack/repos/builtin/packages/py-deeptools/package.py

```diff
@@ -34,10 +35,12 @@ class PyDeeptools(PythonPackage):
 
```

Comment:
For python@3.7 there is a dependency on `py-importlib-metadata` missing (see [pyproject.toml#L33](https://github.com/deeptools/deepTools/blob/3.5.3/pyproject.toml#L33))

Reply:
Added.